### PR TITLE
uiux: allow for changing app's fontsize

### LINKED ISSUE #737: Ability to control font size
I'm on a 4k screen and the fonts on Desktop are a bit small for me. I'd love a way to scale the fonts.

### PR DESCRIPTION
**Pull request by PascalPrecht**
Closes #737

### CODE REVIEW

File: ui/app/AppLayouts/Profile/Sections/AppearanceContainer.qml [hunk 1]
```diff
@@ -92,10 +101,44 @@ Item {
         }
     }
 
+    RowLayout {
+        id: fontSetting
+        anchors.top: themeSetting.bottom
+        anchors.topMargin: 20
+        anchors.left: parent.left
+        anchors.leftMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/AppearanceContainer.qml:109]:
```suggestion
        anchors.leftMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/AppearanceContainer.qml [hunk 2]
```diff
@@ -92,10 +101,44 @@ Item {
...
+        StyledText {
+            text: qsTr("Font size")
+        }
+        ButtonGroup { id: fontSize }
+
+        StatusRadioButton {
+            checked: Style.fontSizeSetting === AppearanceContainer.FontSize.Normal
+            Layout.alignment: Qt.AlignRight
+            ButtonGroup.group: fontSize
+            rightPadding: 15
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/AppearanceContainer.qml:119]:
```suggestion
            rightPadding: Style.current.padding
```

File: ui/app/AppLayouts/Profile/Sections/AppearanceContainer.qml [hunk 3]
```diff
@@ -92,10 +101,44 @@ Item {
...
+            onClicked: {
+                root.updateFontSize(AppearanceContainer.FontSize.Normal)
+            }
+        }
+
+        StatusRadioButton {
+            checked: Style.fontSizeSetting === AppearanceContainer.FontSize.Big
+            Layout.alignment: Qt.AlignRight
+            ButtonGroup.group: fontSize
+            rightPadding: 15
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/AppearanceContainer.qml:130]:
```suggestion
            rightPadding: Style.current.padding
```

**Review by jrainville — COMMENTED** [review]
The code looks good, I'll test in a few to see how it looks.

The only thing missing in the code would be saving the preference in the Qt settings

**Comment by PascalPrecht** [comment]
I recommend NOT merging this one because well... as I've said last week. This definitely needs much more thought. Changing font sizes also affects margins and heights of components and the over all look & feel.

**Comment by jrainville** [comment]
I just tested and apart from a few fields and popups in the Wallet, it looks very good to me. 
I feel like we could merge as is and fix in another PR because the problems are really minor

**Comment by simonricoo** [comment]
@PascalPrecht can we enable app scaling by Scaling Up (CMD+"+") and Scaling Down (CMD+"-")? I think this can save us some hours of work...

**Comment by PascalPrecht** [comment]
Closing in favour of https://github.com/status-im/nim-status-client/pull/1404

**Pull request closed without merging**